nvtabular/workflow.py

```diff
@@ -739,7 +740,7 @@ def apply(
             shuffler = Shuffler(output_path, num_out_files=num_out_files)
         if hugectr_gen_output:
             self.cal_col_names = False
-            huge_ctr = HugeCTR(hugectr_output_path, num_out_files=hugectr_num_out_files,)
+            huge_ctr = HugeCTR(hugectr_output_path, num_out_files=hugectr_num_out_files, output_format=hugectr_output_format,)
```

Comment:
Since the shuffler and huge_ctr will have the same interface after this change, we should consolidate and only have one writer here

Reply:
Do you think we shouldn't keep the binary file generator? Most likely we won't need it after the parquet reader on hugectr side is ready.

Reply:
Long term we should remove - but we should keep it in until hugectr is ready with the parquet reader I think

Reply:
Not sure if I am missing something or I didn't understand properly.

I am seeing that both objects `shuffler` and `huge_ctr` are not mutually exclusive. They can be created at the same time and are passed trough the functions.

Reply:
Lets not worry about this given the upcoming dask